cumulusci.yml

```diff
@@ -1161,6 +1175,14 @@ flows:
                     path: scripts/New_Testing_User.cls
                     apex: createTestingUser();
 
+    hack_to_delete_feature_parameters:
```

Comment:
I'm curious, what are the previous 4 steps for this? Wouldn't there just be the two new tasks?

Reply:
@force2b : I'm not sure if you remember but in the early days of BASIC, it used line numbering. And the way that you made it easy to insert new lines later was to leave gaps between lines. That's what I was thinking. Perhaps it is more confusing than helpful in the context of CumulusCI.

Reply:
I'm heading off to PTO so you can either leave them, change them to 1, 2 or wait for me to get back in a week.

Reply:
Thanks for clarifying.